sync/clients/sync.py

```diff
@@ -94,11 +127,9 @@ def build_auth_request(self) -> httpx.Request:
     def update_access_token(self, response: httpx.Response):
         if response.status_code == httpx.codes.OK:
             auth = response.json()
-            self._access_token = auth["result"]["access_token"]
-            self._access_token_expires_at_utc = auth["result"]["expires_at_utc"]
-            self._set_cached_token(
-                self._access_token, self._access_token_expires_at_utc
-            )
+            access_token = auth["result"]["access_token"]
+            expires_at_utc = dateutil.parser.isoparse(auth["result"]["expires_at_utc"])
```

Comment:
Had to do this because py3.7 can't parse a datetime with timezone with builtin `fromisoformat`

Reply:
boo!